cmd/meroxa/turbine_cli/utils.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/meroxa/cli/log"
 )
 
-const turbineJSVersion = "0.1.7"
+const turbineJSVersion = "0.1.8"
```

Comment:
Definitely not part of this PR, but we should find a way to automate this.

Reply:
Agreed! I think I can borrow some of your work on updating the turbine-go dependency in the CLi here as well